### PR TITLE
Use Artifact Registry for CI images

### PR DESCRIPTION
**Pull request by upodroid**
/cc @kvmware 

Our autobumps are stuck as they don't get bumped anymore because the image locations have changed. I might as well switch to AR properly. Merging this PR will create images on AR and i'll open a follow up PR to change the prowjobs.

### CODE REVIEW

File: images/simple-image.mk [hunk 1]
```diff
@@ -26,14 +26,14 @@
 #  DOCKERBUILDARGS: arguments for `docker build`
 #  DOCKERFILE: use a different Dockerfile, relative to starting subdirectory
 
-REGISTRY   ?= gcr.io
+REGISTRY   ?= europe-docker.pkg.dev
```
Comment (kvmware) [inline images/simple-image.mk:29]: europe?

Comment (upodroid) [reply images/simple-image.mk:29]: typo :D

**Comment by upodroid** [comment]
All the terraform changes have been applied.

**Pull request merged**